skrf/vectorFitting.py

```diff
@@ -112,8 +129,8 @@ def __init__(self, network):
         self.delta_max_history = []
         self.history_max_sigma = []
 
-    def vector_fit(self, n_poles_real=2, n_poles_cmplx=2, init_pole_spacing='lin', parameter_type='S',
-                   fit_constant=True, fit_proportional=False):
+    def vector_fit(self, n_poles_real: int = 2, n_poles_cmplx: int = 2, init_pole_spacing: str = 'lin',
+                   parameter_type: str = 'S', fit_constant: bool = True, fit_proportional: bool = False) -> None:
```

Comment:
in the rest of the code, 's' (lowercase) is used of s-parameters.

Reply:
It does not matter, both lower and upper case works:
https://github.com/scikit-rf/scikit-rf/blob/8c30a735dd1539b26c5d6bfff8baaa81a2dceae1/skrf/vectorFitting.py#L202

Reply:
yes I did see, that was just a matter of uniformization in the documentation

Reply:
Ok. It's changed for lower case 's' in https://github.com/scikit-rf/scikit-rf/pull/505/commits/3c6554b86f46add3171325adb84a037cdbd3b434